src/main/java/edu/hm/hafner/coverage/FileNode.java

```diff
@@ -521,6 +521,17 @@ public FileNode addCounters(final int lineNumber, final int covered, final int m
         return this;
     }
 
+    @CanIgnoreReturnValue
+    public FileNode addCounterIncremental(final int lineNumber, final int covered, final int missed) {
```

Comment:
As addressed in previous statement, my mistake on the previous incorrect information. But because each line read in the coverage file gives N number of covered statements, and M number of missed statements, we keep running tallies of each one and then resolve everything at the end.